draft-ietf-quic-recovery.md

```diff
@@ -521,6 +521,10 @@ is available, or if the network changes, the initial RTT SHOULD be set to 500ms,
 resulting in a 1 second initial handshake timeout as recommended in
 {{?RFC6298}}.
 
+A new path on an existing connection MAY use the delay between sending a
```

Comment:
can you fix this sentence? the path does not use the delay

Reply:
Better?